release/k8s_tests/run_gcs_ft_on_k8s.py

```diff
@@ -51,7 +51,7 @@ def generate_cluster_variable():
 
 def check_kuberay_installed():
     # Make sure the ray namespace exists
-    KUBERAY_VERSION = "v0.5.0"
```

Comment:
Just double checking that we actually want to change it in this PR, since it isn't a doc file.

Reply:
Yes